packages/components/droppable-panel/src/messages.js

```diff
@@ -0,0 +1,10 @@
+import { defineMessages } from 'react-intl';
+
+export default defineMessages({
+  useInputSearch: {
```

Comment:
It feels like this translation doesn't belong to a component library as it's too tied to a more specific behaviour / business logic (hence my other comment above).

Reply:
This input it's a usability thing: When there are more than 50 columns in the table the search input appears

Reply:
Ah I see, this makes sense then. But perhaps we should be more consistent with the namings: here we are calling them "attributes" but in the codebase we call them "columns" (because we know they will be used for the table).

As a standalone component, I feel we should use a more generic name and just call them "items" (or something equivalent), both here and in the codebase.